jib-core/src/main/java/com/google/cloud/tools/jib/registry/ErrorResponseUtil.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.jib.registry;
+
+import com.google.api.client.http.HttpResponseException;
+import com.google.cloud.tools.jib.json.JsonTemplateMapper;
+import com.google.cloud.tools.jib.registry.json.ErrorEntryTemplate;
+import com.google.cloud.tools.jib.registry.json.ErrorResponseTemplate;
+import java.io.IOException;
+import java.util.List;
+
+/** Utility methods for parsing {@link ErrorResponseTemplate JSON-encoded error responses}. */
```

Comment:
Is the `JSON-encoded error responses` intended to be inside the `{@link`?

Reply:
Yeah.  Callers aren't necessarily aware of the class, but its javadocs have a nice description of the object layout.